### ISSUE TITLE
Setting blinds to My position isn't recognized

### ISSUE THREAD

**Issue by phatina** [rstrouse/ESPSomfy-RTS#459]
### Hardware

ESP32

### Firmware version

v2.4.7

### Application version

v2.4.7

### What happened? What did you expect to happen?

If I have my blinds in any other position than "My position" and press My button on Telis 4 remote, they go down to that default position (completely down and 45° tilt). ESPSomfy doesn't pick this command and it thinks that the blinds are open. Am I missing any configuration? Thank you for your time and opinions.

### How to reproduce it (step by step)

```bash
1. Set Somfy blinds to arbitrary position,
2. Press "My" button on a remote
```

**Comment by rstrouse** [comment]
Are you sure your blind motors support the My position if the do not click the Simulate Favorite checkbox.

**Comment by phatina** [comment]
Simulate Favorite Position is unchecked on all of my blinds.

**Comment by rstrouse** [comment]
Do you have a my position that is set in ESPSomfy RTS and if so when you set it did the motor jog?

**Comment by phatina** [comment]
No, I don't have My position set in ESPSomfy yet. Do I need to do it so that ESPSomfy picks up My button press on a remote? Thank you.

**Comment by rstrouse** [comment]
Yes

https://github.com/rstrouse/ESPSomfy-RTS?tab=readme-ov-file#setting-a-favorite

**Comment by llaumgui** [comment]
I have the same issue with my pergola. I don't have state feedback, so I can't associate a favorite position because there’s no "%" position tracking. I use the Up / My / Down buttons for the lights: Up sets the light to full brightness, My to medium brightness, and Down turns the lights off.

I’d like to use the Up / My / Down commands without any position restrictions.

Let me know if you have any suggestions.

**Comment by rstrouse** [comment]
ESPSomfy RTS adds the position feedback to your motor by setting the motor timings.
